Spanner/src/Database.php

```diff
@@ -812,7 +822,9 @@ public function transaction(array $options = [])
         }
 
         // There isn't anything configurable here.
-        $options['transactionOptions'] = $this->configureTransactionOptions();
+        $options['transactionOptions'] = $this->configureTransactionOptions([
```

Comment:
this method will need to be updated for this to work

https://github.com/googleapis/google-cloud-php/blob/973af1c936bbfdc0b4fdfda2ac11ff51beda3ce7/Spanner/src/TransactionConfigurationTrait.php#L145

Reply:
It should be already like that, unless I am mistaken:
https://github.com/googleapis/google-cloud-php/blob/973af1c936bbfdc0b4fdfda2ac11ff51beda3ce7/Spanner/src/TransactionConfigurationTrait.php#L145